site/src/lib/api/model/block.model.ts

```diff
@@ -0,0 +1,124 @@
+import { BlockVariant, JsonObject } from "@blockprotocol/core";
+import { BlockType } from "@blockprotocol/core/dist/esm/types";
+import { Db } from "mongodb";
+
+import localBlocks from "../../../../blocks-data.json" assert { type: "json" };
+import { ExpandedBlockMetadata } from "../../blocks";
+import { User } from "./user.model";
+
+export class Block implements ExpandedBlockMetadata {
```

Comment:
On a base level, model is a javascript object with a few methods. These methods may require different sets of dependencies, which potentially makes the total number of model dependencies quite large. Object methods are not tree-shakeable, so calling a single model method in an API route unnecessary pulls all model dependencies. This makes cold lamda starts pretty slow: https://github.com/vercel/community/discussions/496#discussioncomment-3004219

I would avoid using the concept of models as such and stick with [plain data + transform function] as much as we can. We don't need to remove all models in one go, but at least avoid creating new ones with the same fundamental bundle size issue.

Reply:
I've removed the Block model and replaced it with [this file](https://github.com/blockprotocol/blockprotocol/blob/7d04f497366ca55288508e975b60c9b7d6271b4f/site/src/lib/api/blocks.ts)